test/models/comment_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class CommentTest < ActiveSupport::TestCase
+   test "comment_on_post" do 
```

Comment:
Yup, having placeholders and opening follow-up tickets once it's merged is totally acceptable, but you generally want to make a note of that up front.